packages/react-art/src/__tests__/ReactART-test.js

```diff
@@ -359,58 +357,6 @@ describe('ReactART', () => {
     doClick(instance);
     expect(onClick2).toBeCalled();
   });
-
-  // @gate !enableSyncDefaultUpdates
-  it('can concurrently render with a "primary" renderer while sharing context', () => {
```

Comment:
I don't have context about this test, I'm curious what the output is with the flag. but since the flag is on for a while, seems it's no longer necessary.

Reply:
It yields in different order, but the code path when the @gate doesn't match is very generic just "should fail this test when the condition doesn't match"